protocols/samp.js

```diff
@@ -16,8 +16,10 @@ export default class samp extends Core {
       if (this.isVcmp) {
         const consumed = reader.part(12)
         state.raw.version = this.reader(consumed).string()
+        state.version = state.raw.version
+
+        delete state.raw.version
```

Comment:
By my last comment I wasn't meaning to 'delete' the fields, but rather doing:
```js
state.version = this.reader(consumed).string()
```
instead of
```js
state.raw.version = this.reader(consumed).string()
state.version = state.raw.version
```

Reply:
Ohhhhh, my bad